numpyro/examples/covtype.py

```diff
@@ -0,0 +1,106 @@
+import argparse
+import time
+
+import numpy as onp
+from sklearn.datasets import fetch_covtype
+
+import jax
+import jax.numpy as np
+from jax import random
+
+import numpyro.distributions as dist
+from numpyro.handlers import sample
+from numpyro.hmc_util import initialize_model
+from numpyro.mcmc import hmc
+from numpyro.util import tscan
+
+
+step_size = 0.00167132
+init_params = {"coefs": np.array(
```

Comment:
nit: It is better to use `onp.array` here so `jax.config.update("jax_platform_name", args.device)` can take effect.

Reply:
Interesting. Any idea what happens if the platform is changed later? Do we get an error or does it move the array to the gpu?